src/tools/attack/attack.cpp

```diff
@@ -38,7 +38,7 @@ static QUIC_ADDR ServerAddress;
 static uint64_t TimeoutMs = ATTACK_TIMEOUT_DEFAULT_MS;
 static uint32_t ThreadCount = ATTACK_THREADS_DEFAULT;
 static const char* Alpn = "h3-29";
```

Comment:
Can you update this to `h3` too? Thanks!
```suggestion
static const char* Alpn = "h3";
```